wasm/wasm_source/src/vp_testnet_faucet.rs

```diff
@@ -57,7 +76,10 @@ fn validate_tx(
                                 &ctx.pre(),
                                 &addr,
                             )?;
-                        change >= -max_free_debit.change()
+
+                        token::Amount::from_uint(change.abs(), 0).unwrap()
```

Comment:
should this be the same denom? i.e.
```suggestion
                        token::Amount::from_uint(change.abs(), denom).unwrap()
```

Reply:
I see, thanks!